core/vibes/soul/sections/products-list-section/sorting.tsx

```diff
@@ -66,7 +67,6 @@ function SortingInner({
         });
       }}
       options={options}
-      placeholder={label}
```

Comment:
Are you sure we can just get rid of the `placeholder` prop here? I'm passing it through to `<SelectPrimitive.Value>` so it can be rendered in the button.

Reply:
The `Select` component still accepts a `placeholder` prop, but I removed it because I assumed it would just use the default value for the `placeholder`, which is `Select an item`

Reply:
Can you include an image of how that select looks after these changes? If we are now using `label` to set the `aria-label`, we now need a placeholder prop we can pass from the parent section component no?

Reply:
Yes, exactly. We need to pass in `placeholder` from the parent